src/neo/Helper.cs

```diff
@@ -310,5 +310,90 @@ internal static BigInteger WeightedAverage<T>(this IEnumerable<T> source, Func<T
                 yield return resultSelector(item, weight);
             }
         }
+
+        public static byte[] Concat(this byte[] a, byte[] b)
```

Comment:
This has already been implemented.

https://github.com/neo-project/neo/blob/c0a411474db817d6217c9a3a28a41e67f0947561/src/neo/Helper.cs#L37-L51